companion/src/firmwares/edgetx/yaml_modeldata.cpp

```diff
@@ -108,7 +108,7 @@ static const YamlLookupTable trainerModeLut = {
   {  TRAINER_MODE_SLAVE_JACK, "SLAVE"  },
   {  TRAINER_MODE_MASTER_SBUS_EXTERNAL_MODULE, "MASTER_SBUS_EXT"  },
   {  TRAINER_MODE_MASTER_CPPM_EXTERNAL_MODULE, "MASTER_CPPM_EXT"  },
-  {  TRAINER_MODE_MASTER_SERIAL, "MASTER_BATT_COMP"  },
+  {  TRAINER_MODE_MASTER_SERIAL, "MASTER_SERIAL"  },
```

Comment:
Leave "MASTER_BATT_COMP" in the list for conversion of older settings and add comment "depreciated" but after "MASTER_SERIAL" so encode will write "MASTER_SERIAL"

Reply:
Rodger that... done :)